client/elements/page-selector.html

```diff
@@ -72,17 +72,18 @@
       #drawertoggle {
         z-index: 1;
         color: var(--sc-tertiary-text-color);
+        margin-right: var(--sc-size-md);
       }
 
-      @media screen and (max-width: 450px) {
-        #toolbar_title {
-          font-size: calc(var(--sc-size-md) * var(--sc-skolar-font-scale));
+      @media screen and (min-width: 840px) {
+        #header.drawer-closed {
+          left: 0 !important;
```

Comment:
![important](https://memegenerator.net/img/instances/28142879/important-all-the-css.jpg)